src/components/PostPageDetail/index.js

```diff
@@ -172,10 +170,6 @@ const PostPageDetailIdComponent = (props) => {
       setLoading(false);
     }
   };
-  const updateParentPost = (data) => {
-    setItem(data);
-    updateAllContent(data);
-  };
 
   React.useEffect(() => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateParentPost` function has been removed. If this function was being used elsewhere in the component, make sure its functionality is covered by other parts of the code.
```diff
- const updateParentPost = (data) => {
-   setItem(data);
-   updateAllContent(data);
- };
+ React.useEffect(() => {
+   initial();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->